source/containers.tex

```diff
@@ -18254,7 +18254,7 @@
 
 \pnum
 An integer $r$ is a \defn{rank index} of an index space $S$
-if $r$ is in the range $[0, \text{rank of $S$})$.
```

Comment:
I wouldn't be offended by the status quo, but I appreciate your problems with it and we can make that change. Put an extra space after the closing curly, please, for source readability.

Reply:
After https://github.com/Eelis/cxxdraft-htmlgen/commit/b5293c06e6643c86831c04bc97db37819f58de10 , nested dollar math is no longer an issue.

Reply:
@Eelis Could you open one PR for each commit except for the following? I'm very interested in these changes but I don't think I'm the one that should open PRs for them.
-  [rand.req.eng] Omit superfluous dollar-math wrapping inside \bigoh. [rejected upstream: 1340]
- Use cxxdraft-htmlgen macros to annotate non-C++ code and literal fragments.
- Use monospace font for code in figures.

Reply:
@frederick-vs-ja Honestly, I think most of the commits in my branch address concerns specific to the needs of eel.is/c++draft. Many of these commits could well be a hard sell upstream. If you think there would be value in upstreaming some of these commits, then you're more than welcome to turn them into PRs (I don't need the credit). :)